### PR TITLE
Fixed false positives in switch() with multi-line return statements.

### PR DESCRIPTION
**Pull request by klausi**

### CODE REVIEW

File: CodeSniffer/Standards/Squiz/Tests/ControlStructures/SwitchDeclarationUnitTest.inc [hunk 1]
```diff
@@ -285,3 +285,21 @@ switch ($foo) {
     break;
 
 }
+
+// Correct Multi line breaking statement with return.
+switch ($foo) {
+    case 1:
+    return array(
```
Comment (jian-wu) [inline CodeSniffer/Standards/Squiz/Tests/ControlStructures/SwitchDeclarationUnitTest.inc]: You need one more indent here.

File: CodeSniffer/Standards/Squiz/Tests/ControlStructures/SwitchDeclarationUnitTest.inc [hunk 2]
```diff
@@ -285,3 +285,21 @@ switch ($foo) {
...
+            'whiz',
+            'bang',
+           );
+
+    case 2:
+    return helper_func(
```
Comment (jian-wu) [inline CodeSniffer/Standards/Squiz/Tests/ControlStructures/SwitchDeclarationUnitTest.inc]: One more indent please.

File: CodeSniffer/Standards/Squiz/Tests/ControlStructures/SwitchDeclarationUnitTest.inc [hunk 3]
```diff
@@ -285,3 +285,21 @@ switch ($foo) {
...
+        'whiz',
+        'bang'
+    );
+
+    default:
+    throw new Exception();
```
Comment (jian-wu) [inline CodeSniffer/Standards/Squiz/Tests/ControlStructures/SwitchDeclarationUnitTest.inc]: Same here.

File: CodeSniffer/Standards/Squiz/Tests/ControlStructures/SwitchDeclarationUnitTest.inc (hunk 1 above)
Comment (klausi) [inline CodeSniffer/Standards/Squiz/Tests/ControlStructures/SwitchDeclarationUnitTest.inc]: Nope, this should be fine as is in the Squiz standard.

**Comment by gsherwood** [comment]
Thanks for fix *and the tests*.

@jian-wu The unit test code is correct as per the Squiz standard.

**Pull request merged**